fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -217,6 +202,33 @@ private fun PlaceholderBookmarkImage() {
     )
 }
 
+@Composable
+private fun FaviconFallbackBookmarkImage(
+    bookmark: RecentBookmark,
```

Comment:
Let's only pass the `url` here since that's the only thing that is needed. This makes it easier to refactor and reuse if this comes up elsewhere.